Homework/homework9/test_task3.py

```diff
@@ -0,0 +1,36 @@
+import os
+import shutil
+from pathlib import Path
+from typing import Callable, Optional
+
+from homework9.task3 import universal_file_counter
+
+import pytest
+
+
+@pytest.fixture()
+def dir_path(text1: str, text2: str):
+    path = "Homework/homework9/test_files"
+    os.mkdir(path)
+    with open(os.path.join(path, "file1.txt"), "w+") as file:
+        file.write(text1)
+    with open(os.path.join(path, "file2.txt"), "w+") as file:
+        file.write(text2)
+    yield Path(path)
+    shutil.rmtree(path)
+
+
+@pytest.mark.parametrize(
+    ["text1", "text2", "tokenizer", "token_count"],
+    [
+        ("line1\nline2", "line3", None, 3),
+        ("line 1\nline 2", "line3", str.split, 5),
+        ("", "", None, 0),
+        ("", "", str.strip, 0),
```

Comment:
Just to be sure that we are on the sample page about task requirements could you please add the test: 
```python
("aba\naba", "",  lambda x: x.split("b"),  3)
```

Reply:
Well, I suppose no. As I understood this test should return 4 (two "a" on the first line and two "a" on the second). Or if there is a tokenizer we should ignore \n so it will be "a" "a\na" "a" that equals to 3?

Reply:
> If a the tokenizer is not none, it will count tokens.

IMHO version with  "a" "a\na" "a" better fits to this description.